File: ABC/abc003/abc003_2.py
S = input()
T = input()

string = "atcoder"

ans = True
for i in range(len(S)):
    if S[i] != T[i]:
        if S[i] == "@" and T[i] not in string:
            ans = False
        elif T[i] == "@" and S[i] not in string:
            ans = False
        elif S[i] != "@" and T[i] != "@":
            ans = False

print("You can win" if ans else "You will lose")

File: ABC/abc003/abc003_2.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    string S, T, atcoder = "atcoder";
    cin >> S >> T;
    
    bool ans = true;
    for (int i = 0; i < S.size(); i++) {
        if (S[i] != T[i]) {
            if (S[i] == '@' && atcoder.find(T[i]) == string::npos) ans = false;
            else if (T[i] == '@' && atcoder.find(S[i]) == string::npos) ans = false;
            else if (S[i] != '@' && T[i] != '@') ans = false;
        }
    }
    
    cout << (ans ? "You can win" : "You will lose") << endl;
}